code/utility/src/palindrome/palindrome_check/Readme.md

```diff
@@ -0,0 +1,28 @@
+# Palindrome Checker
+
+The aim is to find if the string is palindrome or not. The string is taken as input from the user.
+
+## Explaination 
+
+- String= "agaga" : Palindrome = YES #The string agaga is same when read from forward or backward. Hence it is a palindrome
+- String= "abccba" : Palindrome = YES
+- String= "rotor" : Palindrome = YES
+- String= "abbaab" : Palindrome = NO
+
+## Algorithm
+
+Palindrome is defined as a string which reads the same backwards as forwards. If there are odd number of characters in the string, we ignore the middle character and check for palindrome. 
+
+Here, after taking string as an input from the user we keep two pointers. One at the starting of the string and the other at the end. We keep on checking if string[start]==string[end] until start<=end .If not, we break the loop and return No else at the end of the loop we print Yes.
```

Comment:
Don't mention pointers, since all languages don't have pointers. Say iterators, or indices.

Starting --> beginning
string[start] == string[end] --> `string[start] == string[end]`
Same for start <= end
Space after period.
we break _from_ the loop.
We return false/true
Otherwise we return true.